week3/homework/src/index.js

```diff
@@ -1,3 +1,65 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const express = require ('express');
+const app = express();
+let todos = require('./todos')
+
+
+
+app.get('/', (req, res) => res.json(todos));
+
+
+app.get('/todos/:id', (req, res) => {
+    let found = todos.find(todo => {
+        return todo.id === parseInt(req.params.id);
+    });
+
+    if (found) {
+        res.status(200).json(found);
+    } else {
+        res.send({ msg: '404 Not found'});
+    }
+});
+
+app.delete('/todos', (req, res) => {
+    let found = todos.find(todo => {
+      return todo.id === parseInt(req.params.id);
+    });
+  
+    if (found) {
+      let targetIndex = todo.indexOf(found);
+      todo.splice(targetIndex, 1);
+    }
+      res.json({ result: "item deleted!"})
+      res.send(200);
+
+  });
```

Comment:
I guess this route supposed to remove all todos, not a single one, so you need to re-write empty array to the data file `./todos.json`.

But if you meant to remove single one, the path would be `'/todos'/:id`, and you need to re-write `todos` object to the file after `todos.splice`